submissions/zalevskyi/dom/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang='en' class='full_screen_height'>
+    <head>
+        <title>Civilization VI - Governors</title>
+        <meta name='viewport' content='width=device-width, initial-scale=1'>
+        <link rel='stylesheet' href='./css/app.css'>
+    </head>
+    <body class='page'>
+        <nav id='nav' class='nav hidden_small'>
+            <ul id='menu' class='menu_list'>
```

Comment:
You can leave it as it is. But next time you can generate this menu from your `WIKI` object